H5项目从Taro 2.0.5 升级到 Taro 3.0.16后，项目编译失败：Module not found: Can't resolve 'nervjs' in project-path/src

<!-- 请不要删除自动生成的 Issue 标签 -->
<!-- 请不要删除自动生成的 Issue 标签 -->
### 相关平台
H5


**浏览器版本: Chrome 86**
**使用框架: React**

### 复现步骤
1、通过Taro 2.0.5 初始化工程；
2、根据Taro next版本升级到Taro 3.0.16，并根据文档修改工程，将工程修改为 react 框架；
3、分别在 微信小程序 和 h5 编译打包

### 期望结果
h5可以正确

### 实际结果
h5编译失败，提示：Module not found: Can't resolve 'nervjs' in ... project-dir/src

### 环境信息
```
Taro v3.0.16


  Taro CLI 3.0.16 environment info:
    System:
      OS: macOS 10.14.4
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 12.6.0 - ~/.nvm/versions/node/v12.6.0/bin/node
      Yarn: 1.9.4 - /usr/local/bin/yarn
      npm: 6.9.0 - ~/.nvm/versions/node/v12.6.0/bin/npm
    npmPackages:
      @tarojs/cli: 3.0.16 => 3.0.16 
      @tarojs/components: 3.0.16 => 3.0.16 
      @tarojs/mini-runner: 3.0.16 => 3.0.16 
      @tarojs/react: 3.0.16 => 3.0.16 
      @tarojs/runtime: 3.0.16 => 3.0.16 
      @tarojs/taro: 3.0.16 => 3.0.16 
      @tarojs/webpack-runner: 3.0.16 => 3.0.16 
      babel-preset-taro: 3.0.16 => 3.0.16 
      eslint-config-taro: 3.0.16 => 3.0.16 
      eslint-plugin-taro: 3.0.16 => 3.0.16 
      react: ^16.10.0 => 16.14.0 
      taro-ui: ^3.0.0-alpha.3 => 3.0.0-alpha.1 
```


---
### 补充信息
1、微信小程序可以打包成功；
2、通过 Taro 3.0.16 初始化一个新项目，然后拷贝代码到新工程可以正确编译打包。

<!-- generated by taro-issues. 请勿修改或删除此行注释 --><!--labels=T-h5,V-3,F-react-->

--- a/packages/taro-service/src/Kernel.ts
+++ b/packages/taro-service/src/Kernel.ts
@@ -302,6 +302,10 @@ export default class Kernel extends EventEmitter {
     }
     if (opts && opts.platform) {
       opts.config = this.runWithPlatform(opts.platform)
+      if (!opts.config.framework) {
+        console.log(helper.chalk.red('请在项目配置中设置 framework 字段：https://taro-docs.jd.com/taro/docs/config'))
+        return
+      }
     }
     await this.applyPlugins({
       name,

